crates/cxx-qt-lib/src/core/qpoint.rs

```diff
@@ -45,7 +45,7 @@ mod ffi {
 }
 
 /// The QPoint struct defines a point in the plane using integer precision.
-#[derive(Debug, Clone, PartialEq)]
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord)]
```

Comment:
So `QPoint` and some others don't have ordering in C++ ( ? ), so if the C++ side doesn't have ordering I don't think we should derive it on the Rust side ? As what does this ordering mean for QPoint ? is (0, 1) < (1, 0) ? as what is "less" ? bottom left in a Cartesian coordinate system ? 

Then for others like QRect vs QRectF they are stored internally in different ways by Qt, so not sure how this could be consistent with derive.

Reply:
I agree with you, I have removed the derives for those types.